api/v1alpha2/types.go

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterSelector specifies a selector for clusters by name or by label with the option to exclude specific clusters.
```

Comment:
There is no option to exclude specific clusters, right?
```suggestion 
// ClusterSelector specifies a selector for clusters by name or by label
```